css/styles.css

```diff
@@ -0,0 +1,302 @@
+html body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Lucida Grande';
+  font-size: 13px;
+}
+
+/* ========= TOP-BAR SECTION */
+
+#main-navbar {
+  display: flex;
+  justify-content: center;
+}
+
+.top-bar {
+  width: 980px;
+  height: 34px;
+  border-top: 1px solid #6a6f73;
+  border-bottom: 1px solid #404549;
+  border-radius: 5px;
+  background: linear-gradient(180deg, #818181 0%, #656565 50%, #545454 50%, #6e6e6e 100%);
+  position: absolute;
+  top: 18px;
+}
+
+.top-bar ul {
+  padding: 0;
+}
+
+.btn, .inner-border {
+  float: left;
+  width: 104px;
+  height: 34px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  border-right: 1px solid #505050;
+}
+
+.btn:hover {
+  background: rgba(0, 0, 0, .3);
+}
+
+.ul-bar, li {
+  margin: 0;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.inner-border {
+  width: 100%;
+  height: 100%;
+  padding-left: 0.5px;
+  padding-right: 0.5px;
+  border-left: .5px solid #7a7a7a;
+  border-right: .5px solid #7a7a7a;
+}
+
+.first-btn>span {
+  border-left: 0;
+}
+
+.first-btn {
+  width: 104px;
+  height: 34px;
+  background: radial-gradient(ellipse at right, rgba(84, 84, 84, 0) 70%, rgba(255, 255, 255, .15) 100%);
+  font-size: 19px;
+}
+
+#search-div {
+  height: 34px;
+  width: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.search {
+  height: 18px;
+  width: 106px;
+  border-radius: 10px;
+  background: rgba(255, 255, 255, .15);
+  border: none;
+  box-shadow: 0 2px 2px inset#353535, 0 .5px 1px #b6b6b6;
+}
+
+#search-div .fa-search {
+  position: absolute;
+  font-size: 12px;
+  top: 36%;
+  left: 88%;
+  color: #fff;
+}
+
+/* MAIN SECTION STYLES*/
+
+#main-section {
+  width: 100%;
+  height: 538px;
+  display: relative;
```

Comment:
The `display` property doesn't accept the `relative` value https://developer.mozilla.org/en-US/docs/Web/CSS/display